requests/exceptions.py

```diff
@@ -25,6 +35,10 @@ def __init__(self, *args, **kwargs):
         super(RequestException, self).__init__(*args, **kwargs)
 
 
+class JSONDecodeError(StandardJSONDecodeError, SimpleJSONDecodeError):
+    """Couldn't decode the text into json"""
+
+
 class InvalidJSONError(RequestException):
```

Comment:
I forgot we had this exception, where do we use it? Should we sub-class this in `JSONDecodeError` as well?

Reply:
I added this exception to sub-class both `JSONDecodeError`s

Reply:
I'm talking about the `InvalidJSONError` exception here (the line I commented on)

Reply:
I think we're still waiting on a response here, @steveberdy.

Reply:
Yes, I can subclass that in `JSONDecodeError` as well.